src/webui/www/private/shareratio.html

```diff
@@ -182,6 +189,16 @@
             <label id="inactiveMinutesLabel" for="setInactiveMinutes">QBT_TR(inactive minutes)QBT_TR[CONTEXT=UpDownRatioDialog]</label>
             <input type="number" id="inactiveMinutes" value="0" step="1" min="0" class="shareLimitInput" aria-labelledby="inactiveMinutesLabel">
         </div>
+        <div style="margin-left: 40px; margin-bottom: 5px;">
+            <label id="actionListLabel" for="action">QBT_TR(Action when the limit is reached)QBT_TR[CONTEXT=UpDownRatioDialog]</label>
+            <select id="action" aria-labelledby="actionListLabel">
+                <option value="-1">Default</option>
+                <option value="0">Stop torrent</option>
+                <option value="1">Remove torrent</option>
+                <option value="3">Remove torrent and its content</option>
+                <option value="2">Enable super seeding for torrent</option>
```

Comment:
The strings should be translatable. Use: `QBT_TR(string)QBT_TR[CONTEXT=UpDownRatioDialog]`